importer/importer.py

```diff
@@ -196,6 +196,7 @@ def get_items(connection,
         print("USING RANDOM SAMPLE OF " + str(short))
     filename = specific_table_name + "_" + utils.get_current_timestamp()
     problem_reports = []
+    wikidata_site = utils.create_site_instance("wikidata", "wikidata")
```

Comment:
You might consider also creating the commons site which you also use in Uploader.

The Wikipedia sites are probably not worth it since they differ too much.